tests/test_testing.py

```diff
@@ -111,16 +111,16 @@ def test_create_environ_cookies_options_method():
 def test_cookies_jar():
     class Foo:
         def on_get(self, req, resp):
-            # NOTE(myuz): In the future we shouldn't change the cookie
+            # NOTE(myusko): In the future we shouldn't change the cookie
             #             a test depends on the input.
             # NOTE(kgriffs): This is the only test that uses a single
             #   cookie (vs. multiple) as input; if this input ever changes,
             #   a separate test will need to be added to explicitly verify
             #   this use case.
-            resp.set_cookie('has_permission', 'true')
+            resp.set_cookie('has_async_permission', 'true')
```

Comment:
Could you explain this renaming though?
Isn't this part used for sync tests as well?

Reply:
Nice catch, that's a typo, will fix it.